### PR TITLE
SPM-1751: db changes to differentiate installable/applicable errata

### PR DESCRIPTION
**Pull request by MichaelMraka**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: manager/controllers/advisories.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ type AdvisoriesDBLookup struct {
 // nolint: lll
 type AdvisoryItemAttributes struct {
 	SystemAdvisoryItemAttributes
-	ApplicableSystems int `json:"applicable_systems" query:"COALESCE(aad.systems_affected, 0)" csv:"applicable_systems" gorm:"column:applicable_systems"`
+	ApplicableSystems int `json:"applicable_systems" query:"COALESCE(aad.systems_installable, 0)" csv:"applicable_systems" gorm:"column:applicable_systems"`
```
Comment (psegedy) [inline manager/controllers/advisories.go:42]: shouldn't we show `aad.systems_installable + aad.systems_applicable` on advisories list page? According to mocks, it should show Affected systems which should be sum of installable and applicable systems

File: manager/controllers/advisories.go [hunk 2]
```diff
@@ -188,15 +188,15 @@ func AdvisoriesListIDsHandler(c *gin.Context) {
 func buildQueryAdvisories(db *gorm.DB, account int) *gorm.DB {
 	query := db.Table("advisory_metadata am").
 		Select(AdvisoriesSelect).
-		Joins("JOIN advisory_account_data aad ON am.id = aad.advisory_id and aad.systems_affected > 0").
+		Joins("JOIN advisory_account_data aad ON am.id = aad.advisory_id and aad.systems_installable > 0").
```
Comment (psegedy) [inline manager/controllers/advisories.go:191]: if we need to show installable + applicable then we need something like 
```suggestion
		Joins("JOIN advisory_account_data aad ON am.id = aad.advisory_id and (aad.systems_installable > 0 or aad.systems_applicable > 0) ").
```

File: manager/controllers/advisories.go [hunk 3]
```diff
@@ -188,15 +188,15 @@ func AdvisoriesListIDsHandler(c *gin.Context) {
...
 		Joins("JOIN advisory_type at ON am.advisory_type_id = at.id").
 		Where("aad.rh_account_id = ?", account)
 	return query
 }
 
 func buildAdvisoryAccountDataQuery(db *gorm.DB, account int) *gorm.DB {
 	query := database.SystemAdvisories(db, account).
-		Select("sa.advisory_id, sp.rh_account_id as rh_account_id, 0 as status_id, count(sp.id) as systems_affected").
+		Select("sa.advisory_id, sp.rh_account_id as rh_account_id, count(sp.id) as systems_installable").
```
Comment (psegedy) [inline manager/controllers/advisories.go:199]: `systems_installable` here is in fact installable + applicable because it is `count(sp.id)`?

Comment (MichaelMraka) [reply manager/controllers/advisories.go:199]: Answering all at once:
This was actually mean just to update schema and fix / rename other stuff to make it consistent.
Real code changes are going to go in next PRs.

**Pull request merged**